backtesting/backtesting.py

```diff
@@ -1163,8 +1174,11 @@ def run(self, **kwargs) -> pd.Series:
 
     def optimize(self,
                  maximize: Union[str, Callable[[pd.Series], float]] = 'SQN',
+                 method: str = 'grid',
+                 max_tries: Union[int, float] = 100,
                  constraint: Callable[[dict], bool] = None,
                  return_heatmap: bool = False,
+                 return_optimization: bool = False,
```

Comment:
I'd prefer `return_heatmap=` and `return_optimization=` be non-exclusive.
* [ ] For `method='skopt'`, `return_heatmap=True` should return a multi-indexed pandas Series with known (evaluated) objective values. I'm quite positive this can be reconstructed from [`OptimizeResult`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html).
* [ ] For both `return_heatmap=True, return_optimization=True`, a _3-tuple should be returned_. (Or in case of `method='grid', return_optimization=True` => error.)

API questions like these are open to discussion, but it's my position that will prevail, so I better be convinced.